ftw/simplelayout/utils.py

```diff
@@ -14,3 +19,14 @@ def get_block_html(block):
                                    IBlockProperties)
     viewname = properties.get_current_view_name()
     return block.restrictedTraverse(viewname)()
+
+
+def get_block_types():
+    platform = getSite()
+    types_tool = getToolByName(platform, 'portal_types')
+
+    dexterity_ftis = filter(
+        IDexterityFTI.providedBy, types_tool.objectValues())
+
+    return filter(
```

Comment:
The [addable_blocks_view](https://github.com/4teamwork/ftw.simplelayout/blob/master/ftw/simplelayout/browser/ajax/addable_blocks_view.py#L30) expects ftis